### ISSUE TITLE
Rollout Script for Values for Globalattributs

### ISSUE THREAD

**Issue by StefanLindenlaub** [PlanningFactory/Solution_Templates_DE#8]
Create a script, that put in the values in the globalattributs

@GerdTautenhahn Which Stored Procedure saves the values of the global attributes? Is it the sx_pf_Post_Product?

**Comment by GerdTautenhahn** [comment]
yes, sx_pf_Post_Product saves the global attributes of the product.

sx_pf_POST_Productline saves the alias name for a GA and the source (name of dropdown list)
